Tasks/VsTest/vstest.ts

```diff
@@ -51,23 +52,22 @@ try {
     if (useNewCollectorFlag && useNewCollectorFlag.toUpperCase() == "TRUE") {
         useNewCollector = true;
     }
-    
+
     var releaseuri = tl.getVariable("release.releaseUri")
-	var context = "CI";
-	if(releaseuri)
-	{
-		context = "CD";
-	}
+    var context = "CI";
+    if (releaseuri) {
+        context = "CD";
+    }
 
     var systemDefaultWorkingDirectory = tl.getVariable('System.DefaultWorkingDirectory');
     var artifactsDirectory = tl.getVariable('System.ArtifactsDirectory');
     var testAssemblyFiles = getTestAssemblies();
```

Comment:
stop using global variables across functions. Only entry function should hold of these variables. pass them to functions